CONVENTIONS.md

```diff
@@ -126,6 +126,19 @@ it is used to manage all elements of the cluster.
 * Every component must remain available to consumers without disruption during upgrades and reconfiguration
   * See the next section for more details
 
+#### High Availability
+
+We focus on minimizing the impact of single node failures by ensuring operators or operands are spread across multiple nodes while maintaining backwards compatibility.
+
+* If the operator or operand wishes to be highly available and can tolerate the loss of one replica, the default configuration for it should be
+  * Two replicas
+  * Hard pod [anti-affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#always-co-located-in-the-same-node) on hostname (no two pods on the same node)
+  * Use the maxUnavailable rollout strategy on deployments (prefer 25% by default for a value)
+* If the operator or operand requires >= 3 replicas
```

Comment:
Should it be mentioned that the minimum requirement is 2 worker nodes which explains why replicas >= 3 mandate soft affinity?

Reply:
I've mentioned it as maintaining backwards compatibility. I can expand on it a bit further if it clarifies things.